### PR TITLE
Add simple profiling for test automation

### PR DESCRIPTION
**Pull request by dhiller**
**What this PR does / why we need it**:

Simple execution time output is now rendered when `KUBEVIRT_PROFILE_MAKE=1`
is used. This is set in `automation/test.sh` by default.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/profile.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+set -e
+
+# use time to record the elapsed time for the step
+shift # get rid of the '-c' supplied by make.
+
+if [[ -z ${KUBEVIRT_PROFILE_MAKE} ]]; then
+    /bin/bash -c "$*"
+else
+    time /bin/bash -c "$*"
```
Comment (rmohr) [inline hack/profile.sh:11]: Would it make sens to just use `date`? I think that would make it easier to detect where we are stalling.

Comment (dhiller) [reply hack/profile.sh]: Done.

**Comment by dankenigsberg** [comment]
I assume the purpose of this PR is to ease debugging. Am I correct @dhiller?

**Comment by dhiller** [comment]
> I assume the purpose of this PR is to ease debugging. Am I correct @dhiller?

Yes, ATM we have problems with slow CI builds and want to know where the most time is consumed.

**Comment by dankenigsberg** [comment]
Thanks, @dhiller. Please include This level of motivation in the primary message of PRs, so that laypersons like me would have a chance to differential a helper PR like this from a PR that that solves something.

**Comment by dhiller** [comment]
Closing this in favor of kubevirt/kubevirt#3609

**Pull request closed without merging**